FiveCalls/FiveCalls/AcknowListView.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  AcknowListView.swift
+//  FiveCalls
+//
+//  Created by Christopher Selin on 1/3/24.
+//  Copyright © 2024 5calls. All rights reserved.
+//
+
+import AcknowList
+import SwiftUI
+
+struct AcknowListView: UIViewControllerRepresentable {
+    func makeUIViewController(context: Context) -> AcknowListViewController {
+        return AcknowListViewController()
```

Comment:
There's a SwiftUI view, but unfortunately it doesn't include the default initializer which loads packages directly by default.